packages/jobs/src/env.ts

```diff
@@ -0,0 +1,19 @@
+import z from 'zod'
+
+import '@latitude-data/env'
+
+const envvars = z.object({
+  NODE_ENV: z.string(),
+  REDIS_HOST: z.string(),
+  REDIS_PASSWORD: z.string().optional(),
+  REDIS_PORT: z.string(),
+  DATABASE_URL: z.string(),
+})
+
+export default envvars.parse(process.env)
+
+declare global {
+  namespace NodeJS {
+    interface ProcessEnv extends z.infer<typeof envvars> {}
+  }
+}
```

Comment:
beautifui :trollface:

Reply:
Not convinced at all. I might switch to t3-env.

Reply:
nein

Reply:
[Their rationale is good](https://env.t3.gg/docs/introduction).

1. Transforms in the env setup, what needs to be a number is a number in the final form.
2. Type safe.

With this solution when you use `process.env.THING` `THING` is not typed when used. With t3-env you can see if this is a number or a string or optional.

Reply:
>  protection against env transformations, which you should never do anyway

Why?